WalletWasabi.Tests/UnitTests/ViewModels/Mocks.cs

```diff
@@ -9,6 +9,7 @@
 using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
 using WalletWasabi.Fluent.ViewModels.Navigation;
 using WalletWasabi.Fluent.ViewModels.Wallets;
+using WalletWasabi.Tests.UnitTests.ViewModels.UIContext;
```

Comment:
Why? In my IDE this is an unnecessary `using`.

Reply:
I renamed that folder/namespace from `UiContext` to `UIContext` because of a conflict with the `UiContext` class.
On my IDE (VS2022) it is necessary. What IDE do you use?

Reply:
I was stupid, nevermind my comments.